### ISSUE TITLE
3.x: Backport hiding default UI controls

### ISSUE THREAD

**Issue by interrrp** [godotengine/godot-proposals#7287]
### Describe the project you are working on

A 2D RPG game with a lot of controls

### Describe the problem or limitation you are having in your project

Would be pretty nice if we can hide the UI controls in the input map:
![image](https://github.com/godotengine/godot-proposals/assets/111111938/92832232-4fa9-438e-93d5-83e38a49f534)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Input map looks cluttered, we should backport the "Show Built-in Actions" toggle from 4 to 3

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Basically the 4.x input map
![image](https://github.com/godotengine/godot-proposals/assets/111111938/90dc91c8-a035-45e4-b7ac-2505ec4b2e66)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No

### Is there a reason why this should be core and not an add-on in the asset library?

AFAIK Assets can't do this

**Comment by Calinou** [comment]
This was already added in the `3.x` branch, and will be present in 3.6: https://github.com/godotengine/godot/pull/69331
